### DISCUSSION TITLE
Can you please make descriptionMaxLines configurable for captions

### DISCUSSION THREAD

**Discussion by DBAKANG-GIT** [igordanchenko/yet-another-react-lightbox#222] [General]
Can you please make descriptionMaxLines configurable for captions

**Comment by igordanchenko** [comment]
Can you clarify your question? `descriptionMaxLines` is already configurable.

**Reply by DBAKANG-GIT** [reply]
Thank you for getting back to me, apologies I wasn't more clear in my previous message, yes it is customizable but I have a set maximum, can this max value be set by myself so for instance on mobile view as I want to have 7 max lines for instance

**Reply by igordanchenko** [reply]
I'm sorry, but I'm still unclear about your use case. 

Have you tried providing the `descriptionMaxLines` prop?

```
<Lightbox
  captions={{ descriptionMaxLines: 7 }}
  // ...
/>
```
